pkg/api/types.go

```diff
@@ -926,6 +914,12 @@ const (
 	// Status code 422
 	StatusReasonInvalid StatusReason = "Invalid"
 
+	// StatusReasonTimeout means that the request could not be completed within the given time.
```

Comment:
Yes. It wasnt clear to me why some of the constants are duplicated in versioned types.
It didnt look like they are needed there and hence I didnt add them.